SDK/RichRelevanceSDK/RCHAPIClient.m

```diff
@@ -249,7 +264,19 @@ - (NSDictionary *)addAuthToParams:(NSDictionary *)requestParams style:(RCHAPICli
 
             break;
         }
+        case RCHAPIClientUserAndSessionParamStyleLongWithAPIKeyInPath: {
+            if (self.clientConfig.userID != nil && self.clientConfig.userID.length > 0) {
+                newParams[kRCHAPICommonParamUserID] = self.clientConfig.userID;
+            }
+
+            if (self.clientConfig.sessionID != nil && self.clientConfig.sessionID.length > 0) {
+                newParams[kRCHAPICommonParamSessionID] = self.clientConfig.sessionID;
+            }
+
+            break;
+        }
         case RCHAPIClientUserAndSessionParamStyleNone: {
+            newParams[kRCHAPICommonParamAPIKey] = self.clientConfig.APIKey;
```

Comment:
#nth Fall through here is fine, but I'd put a `break` to prevent future bugs if someone adds another case and neglects toa dd it.